### PR TITLE
Add a doc for monitoring TiDB cluster

### PR DESCRIPTION
**Pull request by queenypingcap**

### CODE REVIEW

**Comment by queenypingcap** [comment]
@shenli @siddontang @c4pt0r @nolouch PTAL.

File: op-guide/Monitoring_TiDB.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+
+* Using Prometheus to record the detailed information of the various operations in the components, which is called the Metrics interface.
+
+## The component state interface 
+
+You can use this type of interface to monitor the basic information of the component. This interface can act as the interface to monitor Keepalive. In addition, the interface of the Placement Driver (PD) can get the details of the entire TiKV cluster.
+
+### TiDB server
+
+The HTTP interface of TiDB is: http://host:port/status.
```
Comment (siddontang) [inline op-guide/Monitoring_TiDB.md]: use \`http://host:port/status\`

File: op-guide/Monitoring_TiDB.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+* git_hash: the Git Hash of the current TiDB code
+
+### PD server
+
+The API address of PD is: http://${host}:${port}/pd/api/v1/${api_name}
+
+The default port number is: 2379.
+
+See [PD API doc](https://cdn.rawgit.com/pingcap/docs/master/op-guide/pd-api-v1.html) for detailed information about various API names.
+
```
Comment (siddontang) [inline op-guide/Monitoring_TiDB.md]: use \`http://${host}:${port}/pd/api/v1/${api_name}\`

File: op-guide/Monitoring_TiDB.md [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+    * The default account name: admin
+
+    * The password for the default account: admin
+
+2. Click the Grafana logo to open the sidebar menu.
+
+3. Click "Data Sources" in the sidebar.
+
+4. Click "Add data source".
```
Comment (siddontang) [inline op-guide/Monitoring_TiDB.md]: why `\#` here?

File: op-guide/Monitoring_TiDB.md [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+
+5. Specify the data source information:
+
+    * Specify the name for the data source. 
+
+    * For Type, select Prometheus.
+
+    * For Url, specify the Prometheus address.
```
Comment (siddontang) [inline op-guide/Monitoring_TiDB.md]: use toml

File: op-guide/Monitoring_TiDB.md [hunk 5]
```diff
@@ -0,0 +1,183 @@
...
+1. Click the Grafana logo to open the sidebar menu.
+
+2. On the sidebar menu, click "Dashboards" -> "Import" to open the "Import Dashboard" window.
+
+3. Click "Upload .json File" to upload a JSON file. 
+
+4. Click "Save & Open". 
+
+5. A Prometheus dashboard is created.
+
```
Comment (siddontang) [inline op-guide/Monitoring_TiDB.md]: use yaml

**Comment by c4pt0r** [comment]
LGTM

File: op-guide/Monitoring_TiDB.md [hunk 6]
```diff
@@ -0,0 +1,183 @@
+# Monitoring a TiDB cluster
```
Comment (nolouch) [inline op-guide/Monitoring_TiDB.md]: change file name like `monitoring-tidb.md`  to  consistent with others in op-guide

File: op-guide/Monitoring_TiDB.md (hunk 5 above)
Comment (nolouch) [inline op-guide/Monitoring_TiDB.md]: https://github.com/pingcap/docs/pull/60/files#diff-f68c1b6c7f67810b2efd0959890e66e1R191
here need change `honor_labels:true` to `honor_labels: true`

File: op-guide/Monitoring_TiDB.md (hunk 3 above)
Comment (queenypingcap) [inline op-guide/Monitoring_TiDB.md]: To use it as an escape character and avoid conflict with the markdown tags.

**Comment by queenypingcap** [comment]
@shenli PTAL

File: op-guide/monitoring-tidb.md [hunk 7]
```diff
@@ -0,0 +1,240 @@
+# Monitoring a TiDB cluster
+
+Currently there are two types of interfaces to monitor the state of the TiDB cluster:
+
+* Using the HTTP interface to report the component information to the outside, which is called the component state interface.
```
Comment (shenli) [inline op-guide/monitoring-tidb.md]: Using the HTTP interface to get the internal information of a component

**Pull request merged**